docs/remote-k8s-access/README.md

```diff
@@ -0,0 +1,179 @@
+
+The operator stores the remote clients in a cache that persists for the lifetime of the operator and is used across all K8ssandraClusters. 
+
+Let's look at an example ClientConfig:
+
+```yaml
+apiVersion: k8ssandra.io/v1alpha1
+kind: ClientConfig
+meta:
+  name: kind-k8ssandra-1
+spec:
+  contextName: kind-k8ssandra-1
+  kubeConfigSecret: 
+    name: kind-k8ssandra-1-config
+```
+
+The `contextName` field is optional. If omitted the operator will look for a kube context having the same name as the ClientConfig.
+
+## Using a ClientConfig
+Suppose we have two additional ClientConfigs similar to the one in the previous example - `kind-k8ssandra-2` and `kind-k8ssandra-3`. We want to create a K8ssandraCluster with three datacenters, one per Kubernetes cluster.
+
+The K8ssandraCluster manifest would look like this:
+
+```yaml
+apiVersion: k8ssandra.io/v1alpha1
+kind: K8ssandraCluster
+metadata:
+  name: demo
+spec:
+  cassandra:
+    cluster: demo
+    serverVersion: "4.0.1"
+    storageConfig:
+      cassandraDataVolumeClaimSpec:
+        storageClassName: standard
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 5Gi
+    datacenters:
+      - metadata:
+          name: dc1
+        k8sContext: kind-k8ssandra-1
+        size: 3
+      - metadata:
+          name: dc2
+        k8sContext: kind-k8ssandra-2
+        size: 3
+      - metadata:
+          name: dc3
+        k8sContext: kind-k8ssandra-3
+        size: 3
+```
+
+Notice the `k8sContext` property of each element in the `datacenters` array. It should specify the name of a kube context that was provided by a ClientConfig. 
+
+The operator will use the value of `k8sContext` to look up the remote client from the cache and then use it to create the CassandraDatacenter in the remote cluster.
+
+As we can see, the ClientConfig is not used by the K8ssandraCluster. It only needs to know the kube context names.
+
+## Adding or removing a ClientConfig
+As stated earlier, the operator only processes ClientConfigs at startup. If you create or delete a ClientConfig after the operator has already started, it won't have any effect. You have to restart the operator for changes to take effect.
+
+Proceed with caution before deleting a ClientConfig. If there are any K8ssandraClusters that use the kube config provided by the ClientConfig, then the operator won't be able to properly manage them.
+
+## Creating a ClientConfig
+Creating a ClientConfig involves creating the kubeconfig file and secret. This can be error prone to do by hand. Instead use the `create-clientconfig.sh` script which can be found [here](https://github.com/k8ssandra/k8ssandra-operator/blob/main/scripts/create-clientconfig.sh).
+
+**Note:** The operator should already be installed in the remote cluster for which you want to create a ClientConfig.
```

Comment:
Just to make sure I understand: the operator should be _installed_ but – per your words above – not _started_, is that correct?